Taskfile.yml

```diff
@@ -216,7 +216,7 @@ tasks:
         -target-tls
         -postgresql-url='postgres://username@127.0.0.1:5432/ferretdb?search_path='
         -compat-url='mongodb://username:password@127.0.0.1:47018/?tls=true&tlsCertificateKeyFile=../build/certs/client.pem&tlsCaFile=../build/certs/rootCA-cert.pem'
-        -disable-filter-pushdown={{.DISABLE_FILTER_PUSHDOWN}}
+        -disable-all-pushdown={{.DISABLE_ALL_PUSHDOWN}}
```

Comment:
> We should rename "disable filter pushdown" flags to "disable pushdown"

Not to "disable all pushdown"

Reply:
Yeah sorry, noticed that right after requesting review